crates/biome_js_analyze/src/assist/source/use_sorted_keys.rs

```diff
@@ -129,6 +131,26 @@ declare_source_rule! {
     /// };
     /// ```
     ///
+    /// ### `groupByNesting`
+    /// When enabled, groups object keys by their value's nesting depth before sorting alphabetically.
+    ///
```

Comment:
```suggestion
    /// When enabled, groups object keys by their value's nesting depth before sorting alphabetically. 
    /// Simple values (primitives and single-line arrays) are sorted first, followed by nested values (objects and multi-line arrays).
    ///
    /// > Default: `false`
    ///
```

I liked the changeset because it explained what "value's nesting depth" is, which isn't a beginner-friendly term. What do you think of the suggestion?